docs/source/conf.py

```diff
@@ -260,11 +261,22 @@ def setup(app):
     "emcee": ("https://emcee.readthedocs.io/en/latest/", None),
 }
 
-
 # -- Sphinx Gallery Options
 from sphinx_gallery.sorting import FileNameSortKey
 
+# Use pyvista's image scraper for example gallery
+# import pyvista
+# https://github.com/tkoyama010/pyvista-doc-translations/blob/85c835a3ada3a2adefac06ba70e15a101ffa9162/conf.py#L21
+# https://github.com/simpeg/discretize/blob/f414dd7ee7c5ba9a141cb2c37d4b71fdc531eae8/docs/conf.py#L334
+# Make sure off screen is set to true when building locally
+# pyvista.OFF_SCREEN = True
+# # necessary when building the sphinx gallery
+# pyvista.BUILDING_GALLERY = True
+# # Optional - set parameters like theme or window size
+# pyvista.set_plot_theme("document")
+
 sphinx_gallery_conf = {
+    # "image_scrapers": ("pyvista", "matplotlib"),
```

Comment:
Why is this still there?

Reply:
I wanted to keep it, when pyvista fixes this off-screen rendering bug. (So we don't have to look it up again)